lua/starfall/libs_sh/mesh.lua

```diff
@@ -762,6 +775,199 @@ if CLIENT then
 		end
 	end
 
+	--- Starts a new dynamic mesh. If an Mesh object is passed, it will use that mesh instead
+	-- @param mesh Optional Mesh object, mesh to build, can be nil to build a 'dynamic' mesh
+	-- @param prim_type Int, primitive type, see MATERIAL
+	-- @param prim_count Int, the amount of primitives
+	-- @client
+	function mesh_library.begin(mesh, prim_type, prim_count)
```

Comment:
This needs to follow the pattern described in #930. Storing the instructions and applying them later in mesh.End is slower, and it doesn't ensure mesh.End is called.

Reply:
Don't think its bad that it doesn't ensure mesh.End is called since this way a mesh can be worked upon over multiple frames without submitting changes. Also changing it so it follows the pattern described would require a lot of changes which i would rather not do, could close the pr.